### DISCUSSION TITLE
Hexo博客添加live2d看板娘-可换装，增删模型 - 没有秃头基因的燊的博客

### DISCUSSION THREAD

**Discussion by giscus** [yzs020220/giscus-repo#21] [Announcements]
# Hexo博客添加live2d看板娘-可换装，增删模型 - 没有秃头基因的燊的博客

记录如何折腾左下角的看板娘，包括live2d_widget的使用，如何自己修改live2d_api中的模型并直接访问或上传到NPM上再通过CDN加速访问，以及如何调整模型的大小进行适配。 - 没有秃头基因的燊 - 没有秃头基因的燊的博客

https://yzs020220.github.io/posts/41158/

**Comment by LikeRunningWater** [comment]
很好得分享，非常感谢 :D
